### ISSUE TITLE
[BUG]: App stopped playing videos

### ISSUE THREAD

**Issue by doyousmellweed** [yuliskov/SmartTube#4172]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

24.89

### Device Type

Smart TV/Box

### Affected Android

android tv 10

### Steps to reproduce the bug

Normal usage of app

### Actual behavior

Was playing fine until it changed to next video and never loads any video after that. The wheel thing appears, feels like it's going to load, but just keeps turning for ever

### Additional information

Philips 50pug7406
![IMG20241203021552](https://github.com/user-attachments/assets/9d22ca33-c798-4407-9003-52aac4b4e0a4)

**Comment by WBSpitfire** [comment]
Same here. Was working, now SmartTube is dead in the water. Eventually it comes up with an error that it couldn't connect to googlevideo. Wonder if they blocked it again.

**Comment by doyousmellweed** [comment]
> Same here. Was working, now SmartTube is dead in the water. Eventually it comes up with an error that it couldn't connect to googlevideo. Wonder if they blocked it again.

Mine got back to normal. Really don't know what happened...

**Comment by holtolee** [comment]
Same here, android TV freebox player pop

**Comment by tessiselated** [comment]
Same here - it says "Don't close the Player, applying the fix" and then freezes

Sony Bravia Google TV

**Comment by kyleb2112yahoo** [comment]
Mine started doing this again today after the last update had fixed it.  If I log out of my account I can play videos.  It's only when I'm logged in that it does this.

**Comment by daxchar** [comment]
> ### Checklist
> * [x]  I made sure that there are _no existing issues_ - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
> * [x]  I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
> * [x]  I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
> * [x]  This issue contains only one bug.
> 
> ### Affected version
> 24.89
> 
> ### Device Type
> Smart TV/Box
> 
> ### Affected Android
> android tv 10
> 
> ### Steps to reproduce the bug
> Normal usage of app
> 
> ### Actual behavior
> Was playing fine until it changed to next video and never loads any video after that. The wheel thing appears, feels like it's going to load, but just keeps turning for ever
> 
> ### Additional information
> Philips 50pug7406 ![IMG20241203021552](https://private-user-images.githubusercontent.com/190448941/391821654-9d22ca33-c798-4407-9003-52aac4b4e0a4.jpg

**Comment by daxchar** [comment]
Same thing is happening to me :(
I'm probably not as smart as you lot, so I hope it will be fixed soon, fingers crossed.

**Comment by daxchar** [comment]
> Mine started doing this again today after the last update had fixed it. If I log out of my account I can play videos. It's only when I'm logged in that it does this.

I logged out. When trying to play a video, the spinning circle happens and it says "Unknown error. Response code 403"

**Comment by daihashi** [comment]
I'm currently on version 24.89 and I have the same issue. It's been doing this for weeks on my Android TV OS version 10 (Sony Bravia VH1). I try to play a video, and it simply will not play. Sometimes videos will stop playing in the middle of watching a video. If I sign out of my account completely then I can play videos, but obviously I can't access my own feed or subscriptions this way. 

Sometimes if I delete the app data completely then it will start working again when I'm signed in; as in actual app data and not just the app cache. While this works sometimes, it's very inconsistent. Id say 50% of the time doing this resolves it, and the other 50% it achieves absolutely nothing.

**Comment by kyleb2112yahoo** [comment]
I reinstalled the app and that appears to have fixed it for now.  For the past few weeks I've seen a lot of very slow loading of thumbnails whenever I'm logged in.  If I log out everything goes back to being fast, and I've never seen the "won't play video" issue while logged out.

**Comment by daihashi** [comment]
> I reinstalled the app and that appears to have fixed it for now. For the past few weeks I've seen a lot of very slow loading of thumbnails whenever I'm logged in. If I log out everything goes back to being fast, and I've never seen the "won't play video" issue while logged out.

Have you tried just deleting the app data instead of doing a full reinstall?

**Comment by Vegabond24** [comment]
If you clear cache and delete app data then launch it again and sign back into your youtube account it will fix it but mine is to the point where I have to do this nearly daily to continue to use it.

**Comment by daihashi** [comment]
> If you clear cache and delete app data then launch it again and sign back into your youtube account it will fix it but mine is to the point where I have to do this nearly daily to continue to use it.

I'm in the same boat with having to do this almost daily, but it seems to be the most reliable work around for now. I'm not sure if all data is getting corrupted or what, but hopefully devs are able to identify and resolve the issue in the next update.

Anyway, thanks for the reply. I wasn't sure if the  "fix" I was using was specific to me or if it would help others as well. Thanks for confirming.

**Comment by kyleb2112yahoo** [comment]
> > I reinstalled the app and that appears to have fixed it for now. For the past few weeks I've seen a lot of very slow loading of thumbnails whenever I'm logged in. If I log out everything goes back to being fast, and I've never seen the "won't play video" issue while logged out.
> 
> Have you tried just deleting the app data instead of doing a full reinstall?

Yeah, I've cleared app data a lot, but wanted to try the more drastic step.  My thumbnails are still slow. I'm wondering if it's all just YT trying different tricks to trip up SmartTube and NewPipe.

**Comment by simonsmith** [comment]
I also had some success by clearing cache, clearing data, forcing stop and signing in again. Playing videos once again. Previously it was a permanent spinner

**Comment by kyleb2112yahoo** [comment]
SmartTube's not working at all today.  Every day a different problem.   I suspect it's YT cracking down.  Too bad.  When it's working ST is my favorite app ever, but at this rate I'll have to give up on it.

**Comment by marcoscgdev** [comment]
New version released, it is working again 😄

**Comment by doyousmellweed** [comment]
Yeah! Saw it this morning, but forgot to warn. Thank you!

**Comment by Vegabond24** [comment]
If it helps, nearly everytime this issue pops up for like the hour before there are a ton of error 403's with the message "applying fix do not close the player"

**Comment by sk0156408-prog** [comment]
Video not playing lekin search karne par sab video a raha hai sirf play nahi ho raha hai
